clang/lib/Driver/Driver.cpp

```diff
@@ -6311,6 +6471,16 @@ bool clang::driver::isObjectFile(std::string FileName) {
           types::lookupTypeForExtension(Ext) == types::TY_Object);
 }
 
+bool clang::driver::isStaticArchiveFile(const std::string &FileName) {
+  if (!llvm::sys::path::has_extension(FileName))
+    // Any file with no extension should be considered an Object. Take into
+    // account -lsomelib library filenames.
```

Comment:
```suggestion
    // Any file with no extension should be considered an Object or
    // an -lsomelib library filename.
```
(Just to highlight that such libraries aren't some special case)

Reply:
This was a cut-n-paste problem, adjusted comment accordingly.